spec/features/user_signs_up_spec.rb

```diff
@@ -12,18 +12,18 @@
   # * If I don't specify the required information, I am presented with
   #   an error message
 
-  # scenario 'provide valid registration information' do
-    # visit new_user_registration_path
+  scenario 'provide valid registration information' do
+    visit new_user_registration_path
 
-    # fill_in 'Email', with: 'john@example.com'
-    # fill_in 'Password', with: 'password'
-    # fill_in 'Password confirmation', with: 'password'
+    fill_in 'Email', with: 'john@example.com'
+    fill_in 'Password', with: 'password'
+    fill_in 'Password confirmation', with: 'password'
 
-    # click_button 'Sign up'
+    click_button 'Sign up'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.